stream/views/management.py

```diff
@@ -100,3 +131,17 @@ def get(self, request, *args, **kwargs):
             data,
             content_type="application/json",
             safe=False)
+
+
+class StatusView(generic.View):
+    def post(self, request, *args, **kwargs):
+        if not request.is_ajax():
+            raise Http404
+        data = ""
```

Comment:
what is the use of the variable data here?

Reply:
Nothing, na carry over ra siguro when I reused some code lol